practitioner-service/src/test/java/uk/ac/ox/ndph/mts/practitioner_service/repository/PractitionerStoreTest.java

```diff
@@ -24,28 +25,36 @@
     @Mock
     private EntityConverter<org.hl7.fhir.r4.model.Practitioner, Practitioner> fhirToModelConverter;
 
+    private PractitionerStore practitionerStore;
+
+    @BeforeEach
+    void setup() {
+        practitionerStore = new PractitionerStore(repository, modelToFhirConverter, fhirToModelConverter);
+    }
+
     @Test
     void TestSaveEntity_WhenValidEntity_SaveToRepositoryAndReturnGeneratedId() {
-        //arrange
+        // arrange
         Practitioner inputPractitioner = new Practitioner(null, "prefix", "givenName", "familyName", "userAccountId");
         var outputPractitioner = new org.hl7.fhir.r4.model.Practitioner();
         when(modelToFhirConverter.convert(any(Practitioner.class))).thenReturn(outputPractitioner);
         when(repository.savePractitioner(any(org.hl7.fhir.r4.model.Practitioner.class))).thenReturn("123");
 
-        //act
-        PractitionerStore practitionerStore = new PractitionerStore(repository, modelToFhirConverter, fhirToModelConverter);
+        // act
         var result = practitionerStore.saveEntity(inputPractitioner);
 
-        //assert
+        // assert
         assertThat(result, equalTo("123"));
     }
 
     @Test
-    void TestListEntitiesByUserIdentity_WhenListByUserIdentity_ThrowsNotImplementedException() {
-        //act and assert
-        PractitionerStore practitionerStore = new PractitionerStore(repository, modelToFhirConverter, fhirToModelConverter);
-        Assertions.assertThrows(UnsupportedOperationException.class,
-                () -> practitionerStore.findEntitiesByUserIdentity("123"),
-                "Expecting to throw an exception");
+    void TestListEntitiesByUserIdentity_WhenFindByUserIdentity_CallsRepository() {
```

Comment:
This method doesn't assert and is a code-smell in the sonar report.

Reply:
Added an assert